otter/grade/containers.py

```diff
@@ -1,16 +1,15 @@
 """Docker container management for Otter Grade"""
-
-from python_on_whales import docker
 import glob
-import itertools
 import os
-import pandas as pd
 import pickle
-import pkg_resources
 import shutil
 import tempfile
-
 from concurrent.futures import ThreadPoolExecutor, wait
+from typing import Optional
+
+import pandas as pd
+import pkg_resources
+from python_on_whales import docker
```

Comment:
do you mind re-ordering these imports to match the style? sorry for the nit-picking but I'm trying to slowly-but-surely fix a but of mismatched styles in this repo and would like all new contributes not to introduce new style issues.

order should be

* `import ...< as ...>` statements in alphabetical order
* &lt;empty line&gt;
* `from ... import ...` statements _not from within the `otter` package_ in alphabetical order
* &lt;empty line&gt;
* `from ... import ...` statements importing from within the `otter` package.

Thanks!

Reply:
No Problem. That's the fault of pycharm. I'll try to change the settings (: